### PR TITLE
Index as part of ElasticsearchSource for parity with Grafana

### PR DESCRIPTION
**Pull request by elainearbaugh**
- Add index to ElasticsearchSource

since this is how Grafana Elasticsearch data sources work. 

I just edited the migration since it's not deployed on prod yet and it seems better to consolidate migrations, lmk if I should create a new one instead.

### CODE REVIEW

**Comment by elainearbaugh** [comment]
@mahendra removed the URL changes, now this is only adding the index field

**Comment by elainearbaugh** [comment]
trigger build

File: cabot/metricsapp/tests/test_elasticsearch.py [hunk 1]
```diff
@@ -6,7 +6,8 @@ class TestElasticsearchSource(TestCase):
     def setUp(self):
         self.es_source = ElasticsearchSource.objects.create(
             name='elastigirl',
```
Comment (mahendra) [inline cabot/metricsapp/tests/test_elasticsearch.py]: `elastigirl` ??

Comment (elainearbaugh) [inline cabot/metricsapp/tests/test_elasticsearch.py]: it's from the incredibles http://disney.wikia.com/wiki/Helen_Parr
sorta close to elasticsearch.....

**Pull request merged**